src/modals/view-issue-modal.tsx

```diff
@@ -0,0 +1,216 @@
+import { Box, Text, useInput } from "ink";
+import React from "react";
+import type { z } from "zod";
+import { useGetIssueTransitionsQuery } from "../api/get-issue-transitions.query.js";
+import type { issue as issueSchema } from "../api/get-issues.query.js";
+import { useGetPrioritiesQuery } from "../api/get-priorities.query.js";
+import { useGetUsersQuery } from "../api/get-users.query.js";
+import { useTransitionIssueMutation } from "../api/transition-issue.mutation.js";
+import { useUpdateIssueMutation } from "../api/update-issue.mutation.js";
+import { env } from "../env.js";
+import { priorityMap } from "../issue.js";
+import { ADFRenderer } from "../lib/adf-renderer.js";
+import type { TopLevelNode } from "../lib/nodes.js";
+import { openIssueInBrowser } from "../lib/utils/openIssueInBrowser.js";
+import { PaddedText } from "../padded-text.js";
+import { useStdoutDimensions } from "../useStdoutDimensions.js";
+import { SelectLaneModal } from "./select-lane-modal.js";
+import { SelectPriorityModal } from "./select-priority-modal.js";
+import { SelectUserModal } from "./select-user-modal.js";
+
+export const ViewIssueModal = ({
+  issue,
+  onClose,
+}: { issue: z.infer<typeof issueSchema>; onClose: () => void }) => {
+  const { mutate: updateIssue } = useUpdateIssueMutation();
+  const { mutate: transitionIssue } = useTransitionIssueMutation();
+  const { data: _users } = useGetUsersQuery(issue.id); // Preload users
+  const { data: _priorities } = useGetPrioritiesQuery(issue.fields.project.id); // Preload priorities
+  const { data: _transitions } = useGetIssueTransitionsQuery(issue.id); // Preload transitions
+  const [columns, rows] = useStdoutDimensions();
+  const [topOffset, setTopOffset] = React.useState(0);
+
+  const [updateAssigneeModalOpen, setUpdateAssigneeModalOpen] =
+    React.useState(false);
```

Comment:
```suggestion
    useState(false);
```

ditto others with `React.useState`